files/es/learn/css/building_blocks/the_box_model/index.html

```diff
@@ -309,7 +309,7 @@ <h2 id="El_uso_de_display_inline-block">El uso de display: inline-block</h2>
 
 <h2 id="Pon_a_prueba_tus_habilidades">Pon a prueba tus habilidades</h2>
 
-<p>Hemos cubierto mucho terreno en este artículo. ¿Recuerdas la información más relevante? Encontrarás más pruebas para verificar que has retenido esa información en <a href="/en-US/docs/Learn/CSS/Building_blocks/Selectors/Box_Model_Tasks">Test your skills: The Box Model</a>.</p>
+<p>Hemos cubierto mucho terreno en este artículo. ¿Recuerdas la información más relevante? Encontrarás más pruebas para verificar que has retenido esa información en <a href="/en-US/docs/Learn/CSS/Building_blocks/Box_Model_Tasks">Test your skills: The Box Model</a>.</p>
```

Comment:
Muchas gracias @JuanVqz. Tengo una duda. En estos casos quizas sea preferible usar el enlace a la version en espanol y que el fallback se encargue de redireccionarlo a la version en ingles de no existir. Asi cuando se traduzca la pagina no existente no haya que volver a modificar este archivo. Me comentas que opinas para tener en cuenta como tratar estos casos.

```suggestion
<p>Hemos cubierto mucho terreno en este artículo. ¿Recuerdas la información más relevante? Encontrarás más pruebas para verificar que has retenido esa información en <a href="/es/docs/Learn/CSS/Building_blocks/Box_Model_Tasks">Test your skills: The Box Model</a>.</p>
```

Reply:
Si tienes toda la razón! debemos de asegurarnos de usar el fallback **siempre**